index.html

```diff
@@ -59,25 +59,55 @@
     </div>
   </nav>
   <section id="about-us">
```

Comment:
Just to be consistent with #7, can you add the padding and margin needed?

```suggestion
  <section id="about-us" class="p-3 my-4">
```